tsql/TSqlParser.g4

```diff
@@ -1411,52 +1550,52 @@ derived_table
+    | CONVERT '(' convert_data_type=data_type ','convert_expression=expression (',' style=expression)? ')'				#CONVERT
     // https://msdn.microsoft.com/en-us/library/ms189788.aspx
-    | CHECKSUM '(' '*' ')'
+    | CHECKSUM '(' '*' ')'				#CHECKSUM
     // https://msdn.microsoft.com/en-us/library/ms190349.aspx
-    | COALESCE '(' expression_list ')'
+    | COALESCE '(' expression_list ')'			#COALESCE
     // https://msdn.microsoft.com/en-us/library/ms188751.aspx
-    | CURRENT_TIMESTAMP
+    | CURRENT_TIMESTAMP					#CURRENT_TIMESTAMP
     // https://msdn.microsoft.com/en-us/library/ms176050.aspx
-    | CURRENT_USER
+    | CURRENT_USER					#CURRENT_USER
     // https://msdn.microsoft.com/en-us/library/ms186819.aspx
-    | DATEADD '(' ID ',' expression ',' expression ')'
+    | DATEADD '(' ID ',' expression ',' expression ')'	#DATEADD
     // https://msdn.microsoft.com/en-us/library/ms189794.aspx
-    | DATEDIFF '(' ID ',' expression ',' expression ')'
+    | DATEDIFF '(' ID ',' expression ',' expression ')'	#DATEDIFF
     // https://msdn.microsoft.com/en-us/library/ms174395.aspx
-    | DATENAME '(' ID ',' expression ')'
+    | DATENAME '(' ID ',' expression ')'		#DATENAME
     // https://msdn.microsoft.com/en-us/library/ms174420.aspx
-    | DATEPART '(' ID ',' expression ')'
+    | DATEPART '(' ID ',' expression ')'		#DATEPART
     // https://docs.microsoft.com/en-us/sql/t-sql/functions/getdate-transact-sql
-    | GETDATE '(' ')'
+    | GETDATE '(' ')'					#GETDATE
     // https://docs.microsoft.com/en-us/sql/t-sql/functions/getdate-transact-sql
-    | GETUTCDATE '(' ')'
+    | GETUTCDATE '(' ')'				#GETUTCDATE
     // https://msdn.microsoft.com/en-us/library/ms189838.aspx
-    | IDENTITY '(' data_type (',' seed=DECIMAL)? (',' increment=DECIMAL)? ')'
+    | IDENTITY '(' data_type (',' seed=DECIMAL)? (',' increment=DECIMAL)? ')'								#IDENTITY
     // https://msdn.microsoft.com/en-us/library/bb839514.aspx
-    | MIN_ACTIVE_ROWVERSION
+    | MIN_ACTIVE_ROWVERSION				#MIN_ACTIVE_ROWVERSION
     // https://msdn.microsoft.com/en-us/library/ms177562.aspx
-    | NULLIF '(' expression ',' expression ')'
+    | NULLIF '(' expression ',' expression ')'		#NULLIF
     // https://msdn.microsoft.com/fr-fr/library/ms188043.aspx
-    | STUFF '(' expression ',' DECIMAL ',' DECIMAL ',' expression ')'
+    | STUFF '(' expression ',' DECIMAL ',' DECIMAL ',' expression ')'									#STUFF
     // https://msdn.microsoft.com/en-us/library/ms177587.aspx
-    | SESSION_USER
+    | SESSION_USER					#SESSION_USER
     // https://msdn.microsoft.com/en-us/library/ms179930.aspx
-    | SYSTEM_USER
+    | SYSTEM_USER					#SYSTEM_USER
     // https://msdn.microsoft.com/en-us/library/ms184325.aspx
-    | ISNULL '(' expression ',' expression ')'
+    | ISNULL '(' expression ',' expression ')'		#ISNULL
     // https://docs.microsoft.com/en-us/sql/t-sql/xml/xml-data-type-methods
-    | xml_data_type_methods
+    | xml_data_type_methods				#XML_DATA_TYPE_METHODS
```

Comment:
I'm not sure about alternative labels here. @Muhamedali what do you say about that?